### ISSUE TITLE
Video materials in reverse don't behave as expected

### ISSUE THREAD

**Issue by likangning93** [AnalyticalGraphicsInc/cesium#6765]
Video materials synchronized to the clock and then played in reverse don't behave as expected (reverse playback).

In Edge this seems to manifest as segments of the video running forward but ordered backwards.
In Chrome and Firefox this manifests for me as a frozen context.

https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/?src=Video.html

**Comment by mramato** [comment]
This is definitely a regression.  Wonder if it's related to recent auto-play changes in most major browsers.

**Comment by lukesanantonio** [comment]
I did a little testing and it looks like autoplay is definitely an issue for some browsers. I think we should mute the video by default so that it is more likely to start automatically for the demo. See [this blog post on Chrome's implementation](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes)

That being said, I don't think that is what's going on here. I think it has to do with the way `VideoSynchronizer` sets the video's playback rate to a negative number. Turns out the behavior of a negative value isn't the same across browsers. See https://github.com/whatwg/html/issues/2754

For example, the sandcastle demo works as expected in Safari (once the video is manually started). I can also confirm that the context freezes in both Chrome and Firefox, but if you have "Video Overlay" enabled in the demo, you'll find the video keeps playing using the absolute value of `playbackRate`. This confirms the behavior mentioned by the OP of that issue I linked above.

The discussion is pretty interesting. Someone pointed out that the Apple website uses reverse playback to spin the [iMac Pro here](https://www.apple.com/imac-pro/), but I can't find any `<video>` elements on that page today, so who knows if they've changed something. This person claims the site just seeks backwards manually for browsers that don't support reverse playback. I wonder if this is the only way we can make `VideoSynchronization` work again. Of course, I could be missing something, :thinking: I hope seeking backwards manually at higher speeds is smooth enough to look okay.
